openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/RichTextEditorPreviewer.tsx

```diff
@@ -51,6 +52,23 @@ const RichTextEditorPreviewer = ({
               </ul>
             );
           },
+          a: ({ node, children, ...props }) => {
+            let link = '';
+            const href = props.href;
+            if (
+              href?.indexOf('http://') == 0 ||
```

Comment:
Should we add a check with some util function `isValidUrl(href)`?

isValidUrl could be a regex check

Reply:
Okay. I will add the `isValidUrl` util.